packages/framework/src/Component/Router/FriendlyUrl/FriendlyUrlFacade.php

```diff
@@ -96,8 +97,9 @@ public function createFriendlyUrlForDomain($routeName, $entityId, $entityName, $
     /**
      * @param \Shopsys\FrameworkBundle\Component\Router\FriendlyUrl\FriendlyUrl $friendlyUrl
      * @param string $entityName
+     * @param bool $setAsMain
      */
-    protected function resolveUniquenessOfFriendlyUrlAndFlush(FriendlyUrl $friendlyUrl, $entityName)
+    protected function resolveUniquenessOfFriendlyUrlAndFlush(FriendlyUrl $friendlyUrl, $entityName, bool $setAsMain = true)
```

Comment:
sadly, this is a BC break :disappointed: If someone extended the facade and overwrote the method, his application would start failing as the declaration would not be compatible with the parent anymore...could you please rewrite this to be backward compatible? Or maybe retargeting to `9.0` would be ok, however, please, consult it with @LukasHeinz - we wanted to have this fixed in `7.3.2`